docs/getting_started/quickstart.md

```diff
@@ -0,0 +1,58 @@
+# Quickstart
+
+This page explains how to quickly install and set up eZ Platform for development.
+
+## Basic requirements
+
+The recommended setup for using eZ Platform is:
+
+- Linux operating system (Debian, Ubuntu or RHEL / CentOS)
+- PHP 7.2
+    - with following packages: `php-cli`, `php-fpm`, `php-mysql`, `php-xml`, `php-mbstring`, `php-intl`, `php-curl`, `php-gd`
+- MySQL
+
+For more details see [Full requirements](requirements_and_system_configuration.md).
+
+## Installation
+
+To install eZ Platform you need [Composer](https://getcomposer.org/) and git on your system.
+
+``` bash
+composer create-project --keep-vcs ezsystems/ezplatform .
+composer ezplatform-install
+```
+
+For more details see [Install using Composer](install_using_composer.md).
+
+See [Install manually](install_manually.md) to install eZ Platform on a different OS.
+
+!!! tip "Demo site"
+
+    You can alternatively install an example demo site:
+
+    ``` bash
+    composer create-project --keep-vcs ezsystems/ezplatform-demo .
+    ```
+
+    The demo site is only a showcase. **Do not use it as a basis for development**.
+
+## Run eZ Platform
+
+To run eZ Platform on the built-in PHP server (for development only):
```

Comment:
the point of this page is development set up so I would remove `(for development only)`